src/core/utils/common.tsx

```diff
@@ -43,3 +43,16 @@ export const getUserColor = (emailOrName: string) => {
   }
   return USER_COLORS[hash];
 };
+
+/**
+ * Get the portal container, or create one if it doesn't exist.
+ */
+export const getContainer = (containerId: string) => {
```

Comment:
So this is not just for portal anymore. Maybe need to rename and update JSDoc comment

Reply:
updated